sycl/test-e2e/Graph/Threading/submit.cpp

```diff
@@ -1,82 +0,0 @@
-// REQUIRES: cuda || level_zero, gpu
```

Comment:
What happened to this test, is it obsolete now? Couldn't find it mentioned in the description of this PR, or did I overlook something?

Reply:
You are right, I forgot to mention it in the PR description.
This test is no longer valid because concurrent submission is not supported by the implementation. (Indeed it was never valid according to spec...)
I updated the PR description to mention this.